x-pack/libbeat/licenser/elastic_fetcher.go

```diff
@@ -14,12 +14,11 @@ import (
 
 	"github.com/pkg/errors"
 
-	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/outputs/elasticsearch"
 )
 
-const xPackURL = "/_xpack"
+const xPackURL = "/_license"
```

Comment:
hm... would it make sense to be BC compatible and set the URL based on the ES version reported by the ES Client?

Reply:
Both endpoint answer on  7.x series for Elastic Licensed ES so I don't think it's necessary to have it backward compatible. Also we only added this checin in 7.x.